main/ertp/guide/README.md

```diff
@@ -6,23 +6,30 @@ Bitcoin has unspent transactions. Ethereum has account balances. Agoric has ERTP
 
 ERTP is a uniform way of transferring tokens and other digital assets in JavaScript. All kinds of digital assets can be easily created, but importantly, they can be transferred in exactly the same ways, with exactly the same security properties.
 
-Get started with the [ERTP Guide](/ertp/guide/) or [ERTP API](/ertp/api/).
+You are currently reading the guide. There is also an [ERTP API reference](/ertp/api/) available.
 
 ## How It Works
 
 ERTP, or the Electronic Rights Transfer Protocol, is a smart contract framework. Throw away all you know about public key hashes mapped to account balances, because ERTP doesn't use 'em.
 
 ERTP itself doesn't have any concept of cryptography. Instead, it uses object capabilities to enforce access control. Instead of having to prove ownership of a corresponding private key, in the world of object capabilities, if your program has a reference to an object, it can call methods on that object. If it doesn't have a reference, it can't. For more on object capabilities, see [Chip Morningstar's post](http://habitatchronicles.com/2017/05/what-are-capabilities/).
 
+ERTP is the top layer of the Agoric stack. You can find out more about this stack in this introductory video:
+<iframe width="560" height="315" src="https://www.youtube.com/embed/52SgGFpWjsY" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
@barbaraliau  does this kind of iframe work under vuePress? I can't tell from my preview.

Reply:
Yes, iframe will work. As long it is supported in Markdown, it will work with VuePress. It looks good to me.

If for some reason it didn't work, an alternate way is to do it like this (also located in the same file):

```
[![miller-sfbw-erights](https://user-images.githubusercontent.com/150986/59150095-b8a65200-89e3-11e9-9b5d-43a9be8a3c90.png)][watch]

[watch]: https://www.youtube.com/watch?v=YXUqfgdDbr8
```

I'll merge this all in and deploy the updates.